### ISSUE TITLE
readme?

### ISSUE THREAD

**Issue by Ouwen** [kballard/slack-groupme-gateway#1]
Will you be adding a readme with setup instructions?

**Comment by charlesmbeckwith** [comment]
I doubt there will ever be a readme. There's no configuration, but basically you just set the environment vars to your groupme and slack details and then run coffee app.coffee from what I can tell.

**Comment by lilyball** [comment]
Yeah, I'm not doing anything with this project any more. My one use-case for it is obsolete. I'll leave it here indefinitely, and if someone else finds it really useful and wants to maintain it I'd be happy to do that, but otherwise, it was just a quick project for personal use that I threw up on GitHub in case anyone was curious about it.
